### ISSUE TITLE
class vs base attribute confusion in finger tutorial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4717]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#4717|
|Type|enhancement|
|Created|2010-10-27 14:17:57Z|
|Branch|https://github.com/twisted/twisted/tree/finger-doc-4717|

In the finger tutorial:

```
<p>This program has two protocol-factory-TCPServer pairs, which are
both child services of the application.  Specifically,
the <code base="API"
class="twisted.application.service.Service">setServiceParent</code>
method is used to define the two TCPServer services as children
of <code>application</code>, which implements <code base="API"
class="twisted.application.servce">IServiceCollection</code>.  Both
services are thus started with the application.</p>
```

In the code tags with a 'base' and 'class' attribute, the values are switched.

<details><summary>Searchable metadata</summary>

```
trac-id__4717 4717
type__enhancement enhancement
reporter__lvh lvh
priority__lowest lowest
milestone__ 
branch__branches_finger_doc_4717 branches/finger-doc-4717
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1288189077000000 1288189077000000
changetime__1295710085000000 1295710085000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [30561]) Branching to 'finger-doc-4717'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [30562]) Correct markup typo, add news file. refs [#4717](https://github.com/twisted/twisted/issues/4717)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, but I'd make the news fragment a .misc.  Who reading the release notes is going to care about this?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [exarkun](#note_4):
> Looks good, but I'd make the news fragment a .misc.  Who reading the release notes is going to care about this?

Good point, I'll change that and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [30563]) update news file, refs [#4717](https://github.com/twisted/twisted/issues/4717)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [30564]) Merge finger-doc-4717: Corrected markup typo in the finger tutorial.

Author: thijs
Reviewer: exarkun
Fixes: [#4717](https://github.com/twisted/twisted/issues/4717)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
